### PR TITLE
Update release-1.17 branch config.toml for release v1.19

### PR DESCRIPTION
**Pull request by savitharaghunathan**
Updated config.toml for release v1.19

/hold

### CODE REVIEW

File: config.toml [hunk 1]
```diff
@@ -95,39 +95,40 @@ js = [
...
+url = "https://kubernetes.io"
+
+[[params.versions]]
+fullversion = "v1.18.8"
 version = "v1.18"
-githubbranch = "v1.18.0"
+githubbranch = "v1.18.8"
 docsbranch = "release-1.18"
-url = "https://kubernetes.io"
+url = "https://v-18.docs.kubernetes.io"
```
Comment (sftim) [inline config.toml:109]:
```suggestion
url = "https://v1-18.docs.kubernetes.io"
```
I think

Comment (savitharaghunathan) [reply config.toml:109]: OMG. I can't believe I am making mistakes on the same URL twice 🙈 . Fix coming up right away.

Comment (savitharaghunathan) [reply config.toml]: I hope I fixed it right this time.

**Review by sftim — COMMENTED** [review]
/lgtm

**Comment by jimangel** [comment]
/lgtm
/approve
Per discussion [here](https://kubernetes.slack.com/archives/C1J0BPD2M/p1598466973088100?thread_ts=1598447979.070600&cid=C1J0BPD2M), I am manually merging this PR as part of the 1.19 release. 🎉

**Pull request merged**